src/components/Slider/_Slider.scss

```diff
@@ -0,0 +1,183 @@
+$light-primary: hsl(162, 73%, 70%) !default;
+$light-secondary: hsl(208, 7%, 70%) !default;
```

Comment:
I believe these should be added to the _variables.scss file

Reply:
added